ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -156,7 +156,7 @@ class LocalAssemblerData
 
     std::vector<std::vector<double>> _darcy_velocities
         = std::vector<std::vector<double>>(
-            GlobalDim, std::vector<double>(ShapeFunction::NPOINTS));
+            GlobalDim, std::vector<double>(_integration_method.getNumberOfPoints()));
```

Comment:
I am only saying this because all the other members are initialized in the ctor. So the failing clang-3.4 here correlates with bad style. And maybe improving style here (by moving the initialization code) would make clang-3.4 succeed.
Of course, updating clang is also a valid solution.